include/networkit/Globals.hpp

```diff
@@ -23,16 +23,16 @@ using omp_index = int64_t;
 using omp_index = index;
 #endif // _MSC_VER
 
+using coordinate = double;
+
 using count = uint64_t;    ///< more expressive name for an integer quantity
 using node = index;        ///< node indices are 0-based
 using edgeweight = double; ///< edge weight type
 using edgeid = index;      ///< edge id
 
-constexpr index none = std::numeric_limits<index>::max(); ///< value for not existing nodes/edges
 constexpr edgeweight defaultEdgeWeight = 1.0;
 constexpr edgeweight nullWeight = 0.0;
-
-using coordinate = double;
+constexpr index none = std::numeric_limits<index>::max(); ///< value for not existing nodes/edges
```

Comment:
Are these changes necessary?

Reply:
Unfortunately yes ... with clang-format 15 and 16 there is a more aggressive alignment of comments. Without rearranging the include guards and clang-format offend each other on line `#endif // _MSC_VER`. The other fix would be either to change the include guards or disable the comment alignment.